src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
agreed, it is one of new performance check and maybe we should disable it. ( it is faster this way as casecmp is faster then creating new modified string ). @lslezak what do you think?